libgadget/init.c

```diff
@@ -93,7 +93,7 @@ void init(int RestartSnapNum, DomainDecomp * ddecomp)
 #ifdef BLACK_HOLES
         if(RestartSnapNum == -1 && P[i].Type == 5 )
         {
-            BHP(i).Mass = All.SeedBlackHoleMass;
+            BHP(i).Mass = P[i].Mass;
```

Comment:
This is the initial condition and we are setting the initial mass of the blackholes to the seed accretion mass -- your change would have set it directly to the gravity mass. I think this is reasonable.

Reply:
Could you add a comment `/*  Watchout: Gadget=3 sets this to the seed mass. */

Reply:
Right - we never actually do this, because we never have BHs in the ICs. I considered just removing it entirely and saying that the person putting BHs in the ICs should specify a mass for them, but decided against it. Will add a comment.

Reply:
We probably don't even read BHP(i).Mass from IC files..

Reply:
Ah yes probably true.